openadapt/utils.py

```diff
@@ -439,13 +438,15 @@ def plot_performance(recording_timestamp: float = None) -> None:
     type_to_count = Counter()
     type_to_timestamps = defaultdict(list)
 
+    from openadapt import crud
+
     if not recording_timestamp:
         # avoid circular import
-        from openadapt import crud
-
         recording_timestamp = crud.get_latest_recording().timestamp
     perf_stats = crud.get_perf_stats(recording_timestamp)
     perf_stat_dicts = rows2dicts(perf_stats)
+    mem_stats = crud.get_memory_stats(recording_timestamp)
+    mem_stat_dicts = rows2dicts(mem_stats)
```

Comment:
Is this necessary? Why not just access `mem_stats` directly?

Reply:
fixed in [35e9de1](https://github.com/MLDSAI/OpenAdapt/pull/270/commits/35e9de1cb44a9ec16dc36963019c6aed55a1ade2)